app/Http/Kernel.php

```diff
@@ -14,11 +14,14 @@ class Kernel extends HttpKernel
      * @var array
      */
     protected $middleware = [
+        \App\Http\Middleware\TrustProxies::class,
+        \Illuminate\Http\Middleware\HandleCors::class,
         \App\Http\Middleware\CheckForMaintenanceMode::class,
+        // FIXME: Find the missing preventrequestsduringmaintenance class
```

Comment:
Non blocking, but you're away of this FIXME? Is this planned to be fixed later(tm) or in this PR? :)

Reply:
A bit of both: I thought it'd be nice to do in the future™️, while I also forgot of its existance.

I've checked where it went, and this turned out to be "technical debt" (of the lower order) from a rename that happened from Laravel 7.x to 8.x. I've renamed it and pushed the change.